js&css/web-accessible/init.js

```diff
@@ -34,6 +34,9 @@ ImprovedTube.observer = new MutationObserver(function (mutationList) {
 			for (var j = 0, k = mutation.addedNodes.length; j < k; j++) {
 				ImprovedTube.childHandler(mutation.addedNodes[j]);
 			}
+			for (const node of mutation.removedNodes){
```

Comment:
we might should fix our other playlist buttons like this too @MAZ01001  👍

Reply:
Yes, or really every element that gets inserted into the DOM

That's how I wrote this playlist button:

1. One function that only creates the button (as an object) and returns it (does not insert it into the DOM)
2. and another that checks if it should add a button to the current page (and adds it via 1.) or update any existing buttons

Then 2. can be called anytime since it checks for itself if it should insert something to the DOM, so I added it to every page where a button _might_ be needed (in the `ImprovedTube.childHandler`) and whenever any existing buttons got deleted by YT (the for-loop you highlighted).